crates/aptos/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 All notable changes to the Aptos CLI will be captured in this file. This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and the format set out by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Unreleased
+
+## [6.0.0] - 2025/01/13
```

Comment:
Isn't a comment here missing the Move 2 is now the default?

Reply:
It is added below